src/backend/config.ts

```diff
@@ -323,7 +324,9 @@ class GlobalConfigV0 extends GlobalConfig {
       winePrefix: isWindows ? '' : defaultWinePrefix,
       wineVersion: defaultWine,
       enableEsync: true,
-      enableFsync: true
+      enableFsync: true,
+      eacRuntime: isLinux,
```

Comment:
Probably for native linux games this should be false aswell.

Reply:
Anti cheat runtime settings are only applied for non native games, so it doesn't matter.